docs/javascript.md

```diff
@@ -88,6 +88,32 @@ Takes these options:
 
 Full documentation on how to design multivariate tests, use the data in GA and construct hypothesis tests is on its way soon.
 
+
+## Reporting to Google Content Experiments
+The toolkit includes a library for multivariate testing that is capable of reporting data into Google Content Experiments. 
```

Comment:
I think it would be helpful to link here to some documentation about Google Content Experiments (eg https://developers.google.com/analytics/devguides/platform/experiments-overview )